Browser_IDE/editorMain.js

```diff
@@ -769,23 +777,32 @@ async function runProgram(){
         let compilableFiles = await findAllCompilableFiles();
         let sourceFiles = await findAllSourceFiles();
         if (compilableFiles.length == 0) {
+        	currentNotification.deleteNotification(); 
             displayEditorNotification("Project has no source files! In a "+activeLanguage.name+" project, valid source files end with:</br><ul>"+
                 activeLanguage.compilableExtensions.map((s)=>"<li>."+s+"</li>").join("")+"</ul>",
-                NotificationIcons.ERROR, -1
+                NotificationIcons.ERROR,-1
             );
             return;
         }
 
         let compiled = await currentCompiler.compileAll(await Promise.all(compilableFiles.map(mapBit)), await Promise.all(sourceFiles.map(mapBit)), reportCompilationError);
 
+		currentNotification.deleteNotification();
+
         if (compiled.output != null) {
             executionEnviroment.runProgram(compiled.output);
         } else {
-            displayEditorNotification("Project has errors! Please see terminal for details.", NotificationIcons.ERROR);
+            displayEditorNotification(
+			"Project has errors! Please see terminal for details.",
+			NotificationIcons.ERROR,-1
+			);
         }
     }
     catch (err) {
-        displayEditorNotification("Failed to run program!<br/>"+err.toString(), NotificationIcons.ERROR);
+        displayEditorNotification(
+		"Failed to run program!<br/>"+err.toString(),
+		NotificationIcons.ERROR,-1
+		);
```

Comment:
I think we should fix the indentation of all the newly added lines to use spaces.

Reply:
Hi @daosgava,
Thank you so much for the helpful feedback! I've made the suggested changes, including fixing the indentation to use spaces and moving the ternary operation outside the function calls for better readability.Thanks again for your review!